docs/api-table.js

```diff
@@ -1464,6 +1487,31 @@ export const elements = {
     ],
   },
 
+  Select: {
+    required: [],
+    props: [
+      ['auto-focus', 'boolean', 'false', 'Whether the element should receive focus on render.'],
+      [
+        'invalid',
+        'boolean',
+        'false',
+        'Renders the field in an invalid state. Often paired together with `hint` to provide feedback about the error.',
+      ],
+      ['always', 'boolean', '', 'Whether to always show hint.'],
+      ['hint', 'string', '', 'The content to display as the help text.'],
+      ['label', 'string', '', 'The content to display as the label.'],
+      ['name', 'string', '', 'The name of the select element, used when submitting an HTML form.'],
+      ['optional', 'boolean', '', 'Whether to show optional text.'],
```

Comment:
I know it's copied from @warp-ds/elements props comments, but now that I read it, it sounds a bit confusing, as if there was some text that was optionally visible 😄 Maybe something like this would be better?
```suggestion
      ['optional', 'boolean', '', 'Mark label to indicate that this input is optional'],
```
Based on [React TextFieldProps](https://github.com/warp-ds/react/blob/next/packages/textfield/src/props.ts#L74)

Reply:
Updated in this commit: [eef7e63](https://github.com/warp-ds/tech-docs/pull/216/commits/eef7e638298d392830bfc81184eb31c19f867b0a)